### DISCUSSION TITLE
I don't understand the suppression of useToastContainer and useToast

### DISCUSSION THREAD

**Discussion by amarant** [fkhadra/react-toastify#1188] [General]
I don't understand the suppression of useToastContainer and useToast, I used them to have the same toast API to use as a banner, so not floating but integrated in the UI. 
It was very simple to use and I didn't need any deep dive in react-toastify source code, it was a good decision to expose them in the first place, and there is no lesson to learn here.

**Comment by fkhadra** [comment]
@amarant sorry for the trouble. Would you be able to share how you are currently using them? Couldn't you achieve the same by replacing the style?
